pkg/operator/ceph/cluster/mon/spec.go

```diff
@@ -183,9 +183,9 @@ func (c *Cluster) makeMonPod(monConfig *monConfig, canary bool, PVCName string)
 // Init and daemon containers require the same context, so we call it 'pod' context
 
 // PodSecurityContext detects if the pod needs privileges to run
-func PodSecurityContext() *v1.SecurityContext {
+func PodSecurityContext(force bool) *v1.SecurityContext {
```

Comment:
Only OSDs need to force to run privileged, right? Should we just have a helper method `privilegedContext()` that we only call in the OSD package? Then no need to modify all the other callers of this method.

Reply:
Or if you want to keep it this way, maybe rename the param to `forcePrivileged`.

Reply:
Yes, I feel that it might go broader than the OSD at some point, ok I'll rename to `forcePrivileged`.

Reply:
What about having a `PrivilegedContext()` method next to the `PodSecurityContext()`? So the OSD controller would call the new method and the the controllers would remain unchanged. Mmost callers use `!ForcePrivileged`, which is hard to understand IMO.